src/Compilers/CSharp/Portable/Binder/LocalBinderFactory.cs

```diff
@@ -224,6 +224,11 @@ public override void VisitLocalFunctionStatement(LocalFunctionStatementSyntax no
                 var oldMethod = _containingMemberOrLambda;
                 _containingMemberOrLambda = match;
 
+                foreach (var parameter in node.ParameterList.Parameters)
+                {
+                    Visit(parameter.Default);
```

Comment:
@gafter this isn't something we should delay.  There is an incossitsency here we need to undercstand before we can merge a fix.